### DISCUSSION TITLE
open discussion

### DISCUSSION THREAD

**Discussion by janryWang** [community/community#2870] [Discussions] [answered]
github.com/alibaba/formily need a discussion feature

**Comment by tomkerkhove** [comment, marked as answer]
You can enable this yourself on your open-source repo; go to Settings > Features:
![image](https://user-images.githubusercontent.com/4345663/101598002-c8975b80-39f7-11eb-8ba6-8e977a39c602.png)

Private repos are not supported and tracked on https://github.com/github/feedback/discussions/2853
